### ISSUE TITLE
[1.22.0] Avatars in @ mentions not loaded with Gitea in subpath

### ISSUE THREAD

**Issue by FrostKiwi** [go-gitea/gitea#31546]
### Description

Regression by updating from `1.21.5` to `1.22.0`: Avatars are invisible and have broken URLs when typing an @ mention, see screenshot below.

Gitea is hosted as a sub-path. Not `domain.com` but `domain.com/git`
`app.ini` is configured to handle this:
```
[server]
SSH_DOMAIN = domain.net/git
DOMAIN = domain.net/git
HTTP_PORT = 3000
ROOT_URL = https://domain.net/git/
```

There have been a multitude of issues from time to time with Gitea being configured as a sub-path, with various files either missing or having a doubled sub path in their path and thus being unreachable.

### Gitea Version

1.22.0

### Can you reproduce the bug on the Gitea demo site?

No

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/60887273/9881b6a2-0748-48f2-808c-bbdd06ac9b8a)

### How are you running Gitea?

Installed via Binary

### Database

None

**Comment by wxiaoguang** [comment]
It has been fixed in 1.22-nightly and 1.22.1

**Comment by yp05327** [comment]
As this has been fixed, I will close this issue. Feel free to reopen it if you have any questions.
